src/server/config.ini

```diff
@@ -206,7 +206,7 @@
   # partition guardian is used to keep partitions healthy. 
   # e.g. if max_replicas_in_group = 3, one primary and two secondaries must be kept for each partition 
   partition_guardian_type = partition_guardian
-  replica_assign_delay_ms_for_dropouts = 600000
+  replica_assign_delay_ms_for_dropouts = 300000
```

Comment:
I think it's better to keep replica_assign_delay_ms_for_dropouts, staleness_for_commit, group_check_interval_ms, checkpoint_interval_seconds, fd_lease_seconds, fd_grace_seconds and log_private_reserve_max_time_seconds just as what they have been in `config.ini` ? All of them have been long running on production environments. I'm afraid that once they are changed, maybe there would be some problems for the production environment. We can change the default values in code to be consistent with what they are in `config.ini`.

Reply:
OK, I've reverted these changes and updated the default values in code.